File: Calculator.java
import java.util.Scanner;

public class Calculator {
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        double number1,number2;
        System.out.println("Enter Number1:");
        number1=input.nextDouble();
        System.out.println("Enter Number2:");
        number2=input.nextDouble();
        System.out.printf("Available operations:%n1-Addition%n2-Subtraction%n3-Multiplication%n4-Division%n%n");
        System.out.printf("Choose an operation:");
        int choice=input.nextInt();
        if(choice==1)
        {
            System.out.printf("Additon of "+number1+" "+number2+"%n");
            System.out.println(number1+" + "+number2+" = "+(number1+number2));
        }
        else if(choice==2)
        {
            System.out.printf("Subtraction of "+number1+" "+number2+"%n");
            System.out.println(number1+" - "+number2+" = "+(number1-number2));
        }
        else if(choice==3)
        {
            System.out.printf("Multiplication of "+number1+" "+number2+"%n");
            System.out.println(number1+" * "+number2+" = "+(number1*number2));
        }
        else if(choice==4)
        {
            System.out.printf("Division of "+number1+" "+number2+"%n");
            System.out.println(number1+" / "+number2+" = "+(number1/number2));
        }
        else
        {
            System.out.println("Can't Perform that operation");
        }
        input.close();
    }
}

File: Calculator.cpp
#include<iostream>
using namespace std;

int main()
{
    int choice;
    float n,m;
    cout<<"Enter number1:";
    cin>>n;
    cout<<"Enter number2:";
    cin>>m;
    cout<<"Available Operations:\n1-Addition\n2-Subtraction\n3-Multiplication\n4-Division\n";
    cout<<"Enter your choice:";
    cin>>choice;
    switch(choice)
    {
        case 1: cout<<"Addition of "<<n<<","<<m<<" is "<<(n+m)<<endl;
            break;
        case 2: cout<<"Subtraction of "<<n<<","<<m<<" is "<<(n-m)<<endl;
            break;
        case 3: cout<<"Multiplication of "<<n<<","<<m<<" is "<<(n*m)<<endl;
            break;
        case 4: if(m==0)
                    cout<<"Division Can't be done.\n";
                else
                    cout<<"Division of "<<n<<","<<m<<" is "<<(n/m)<<endl;
            break;
        default: cout<<"Operation Can't be performed.\n";
    }
}